Shoot/www/js/app.js

```diff
@@ -9,17 +9,17 @@ angular.module('starter', ['ionic', 'starter.controllers', 'starter.services'])
 
 .run(function ($ionicPlatform) {
   $ionicPlatform.ready(function () {
-    // more info on how to change these have a look at: 
+    // more info on how to change these have a look at:
     //      https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md
     oauth2.addGoogle({
       name: 'gplus',
       settings: {
         //replace with your own if you want to...
-        clientId: '241956090675-gkeh47arq23mdise57kf3abecte7i5km.apps.googleusercontent.com',
+        clientId: '<your client secret goes here.apps.googleusercontent.com>',
```

Comment:
you don't have to change this setting it's the same as here https://github.com/aerogear/aerogear-ios-cookbook/blob/master/Shoot/Shoot/ViewController.swift#L141